__tests__/app.test.js

```diff
@@ -148,6 +148,71 @@ describe("GET /api/articles", () => {
     })
   })
 })
-
+describe("GET /api/articles/:article_id/comments", () => {
+  test ("200: returns an array of all comment objects for a parametric request to an article via article_id, with a single specific comment", () => {
+    return request(app)
+    .get("/api/articles/6/comments")
+    .expect(200)
+    .then(({body})=>{
+      const {comments} = body
+      expect(comments).toEqual([{
+        body: "This is a bad article name",
+        votes: 1,
+        author: "butter_bridge",
+        article_id: 6,
+        created_at: "2020-10-11T15:23:00.000Z",
+        comment_id: 16
+      }])
+    })
+  })
+  test("200: checks a parametric request to an article with multiple known number of comments (2) for correct keys", () => {
+    return request(app)
+    .get("/api/articles/5/comments")
+    .expect(200)
+    .then(({body})=>{
+      const {comments} = body
+      console.log(JSON.stringify(comments, null, 2))
+      if (comments.length === 2){
```

Comment:
This should be an assertion so that the test fails if the expected number of comments is not retrieved